Config/RDB Info.txt

```diff
@@ -1,336 +1,336 @@
-﻿########################
```

Comment:
He configured his Git server to receive the files in their exact binary form.
Somehow this has caused him to see LF instead of CRLF line breaks.

Maybe he could be right, but the problem with merging this pull request is that it replaces every line in all the files, making Git Blame and other features pretty easily thrown off.